lms/djangoapps/teams/api.py

```diff
@@ -278,3 +278,38 @@ def get_team_for_user_course_topic(user, course_id, topic_id):
             membership__user__username=user.username,
             topic_id=topic_id,
         ).first()
+
+def get_team_anonymous_user_ids(team_id):
+    """ Get the anonymous user IDs for members of a team, used in team submissions
+
+        Returns:
+            (Array) User IDs, sorted to remove any correlation to usernames
+    """
+    team = None
+
+    try:
+        team = CourseTeam.objects.get(
+            team_id=team_id
+        )
+    except CourseTeam.DoesNotExist:
+        return None
+    except CourseTeam.MultipleObjectsReturned:
```

Comment:
So, if this *should* be unreachable in a reasonable world, should we be making a filter selection and considering it a successful request? Why not fail, as this is by comment an impossible situation

Reply:
We can avoid this by passing team info directly.